web/src/components/storage/VolumeForm.jsx

```diff
@@ -70,12 +71,39 @@ const SizeUnitFormSelect = ({ units, ...formSelectProps }) => {
  * @param {Array<import(~/clients/storage).Volume>} props.volumes - a collection of storage volumes
  * @param {object} props.formSelectProps - @see {@link https://www.patternfly.org/components/forms/form-select#props}
  * @returns {ReactComponent}
- */
-const MountPointFormSelect = ({ volumes, ...formSelectProps }) => {
+*/
+
+const MountPointFormSelect = ({ volumes, ...selectProps }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [chosen, setchosen] = useState(false);
```

Comment:
Use camelCase. I.e., `setChosen` instead of `setchosen`.

```suggestion
  const [chosen, setChosen] = useState(false);
```

Do not forget updating other lines where using `setchosen`.